lib/internal/process/next_tick.js

```diff
@@ -86,8 +89,9 @@ function setupNextTick() {
     _runMicrotasks();
 
     if (tickInfo[kIndex] < tickInfo[kLength] ||
-        emitPendingUnhandledRejections())
+        emitPendingUnhandledRejections()) {
```

Comment:
Style choice, I think we tend to use braces for multi-line conditionals from what I've generally seen, this is just one "offender" I ran across while working in this file.